docs/commands/lint.md

```diff
@@ -22,19 +22,19 @@ redocly lint --version
 
 ## Options
 
-Option | Type | Description
--- | -- | --
-apis | array | Array of API definition filenames that need to be linted. See [the Apis section](#apis) for more options.
---config | string | Specify path to the [configuration file](#custom-configuration-file).
---extends | array | [Extend a specific configuration](#extend-configuration) (defaults or config file settings).
---format | string | Format for the output.<br />**Possible values:** `codeframe`, `stylish`, `json`, `checkstyle`, `codeclimate`, `summary`.
---generate-ignore-file | boolean | [Generate ignore file](#generate-ignore-file).
---help | boolean | Show help.
---lint-config | string | Specify the severity level for the configuration file. <br/> **Possible values:** `warn`, `error`, `off`. Default value is `warn`.
---max-problems | integer | Truncate output to display the specified [maximum number of problems](#max-problems).
---skip-preprocessor | array | Ignore certain preprocessors. See the [Skip preprocessor or rule section](#skip-preprocessor-or-rule) below.
---skip-rule | array | Ignore certain rules. See the [Skip preprocessor or rule section](#skip-preprocessor-or-rule) below.
---version | boolean | Show version number.
+| Option                 | Type     | Description                                                                                                                        |
+| ---------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| apis                   | [string] | Array of API definition filenames that need to be linted. See [the Apis section](#apis) for more options.                          |
```

Comment:
Should this be `api` and not an array if we only accept a single argument? And the description perhaps making it more clear that it's just a string.

Reply:
It accepts multiple apis, so I guess the description is correct.

Reply:
It accepts multiple apis, so I guess the description is correct.

Reply:
I think it is more correct to say that we accept one API argument, supplied multiple times. So the usage should be something like:

```
redocly lint [options ...] apis ...
```

If there is a space in it, it's a new input parameter. At least, that's what I'd expect from using other CLI tools.

Reply:
I realised that this should be a wider conversation, beyond the scope of this pull request.